framework/src/media/streaming/player.cpp

```diff
@@ -27,14 +27,18 @@
 #include "player.h"
 #include "../utils/internal_defs.h"
 
+namespace media {
```

Comment:
Is there any specific reason why changed c to cpp?

Reply:
I want to use audio type `audio_type_t` in `MediaTypes.h` instead of self defined enumeration.
I remember you mentioned about this before.
It's impossile to include `MediaTypes.h` to .c file due to namespace.
player.h includes MediaTypes.h and player.c includes player.h -> build error.
so I change .c to .cpp ...